python/lsst/sdm_schemas/schemas/dp02_dc2.yaml

```diff
@@ -16,85 +16,117 @@ tables:
     datatype: long
     description: Unique id. Unique ObjectID
     ivoa:ucd: meta.id;src;meta.main
+    tap:principal: 1
```

Comment:
Not in this PR, but the description "Unique id. Unique ObjectID" above is a little repetitive.